tasks/configure.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Set VerneMQ nofile Limits
+  become: true
+  lineinfile:
+    dest: /etc/default/vernemq
+    create: yes
+    regexp: "^ulimit -n \\d+"
+    line: "ulimit -n {{ sansible_vernemq.nofile }}"
+  notify: Restart VerneMQ
+  when: sansible_vernemq.nofile is defined
+
+- name: Set VerneMQ nofile Limits.conf
+  become: true
+  lineinfile:
+    dest: /etc/security/limits.conf
+    create: no
+    regexp: "^\\*\\s+{{ item }}\\s+nofile\\s+\\d+"
+    line: "* {{ item }} nofile {{ sansible_vernemq.nofile }}"
```

Comment:
I don't think we need to change limits.conf since we have the direct ulimit call in the default file.

Reply:
I originally didn't have this (only /etc/default/vernemq), but it throws error messages when not administered through the init script directly, and fails the Travis tests.

Reply:
OK no worries at all, in that case is there a specific user for VerneMQ that we can target here?

Reply:
I couldn't get the Travis test to work with anything else than *.  VirtualBox works.